packages/web-components/src/components/ic-text-field/ic-text-field.tsx

```diff
@@ -338,6 +343,15 @@ export class TextField {
     return this.rows > 1;
   };
 
+  /**
+   * @internal Emitted when the validationInlineInternal is `true`
+   */
+  @Event() getValidationText: EventEmitter<IcValueEventDetail>;
```

Comment:
How is this intended to be used? Is it for announcing the validation text to screen readers? I can't see it referenced anywhere else.

Reply:
The intention is that this event will be used in the pagination bar so that the validation text can be used in the tooltip above the go to page part of the pagination bar (see screenshot 1).

Screenshot 2 shows an example of a listen event that can be placed in pagination bar. I think this approach will be particularly useful as I had issues trying to get the validation text using querySelector when I was initially testing the pagination bar.

**Screenshot 1:**
<img width="329" alt="Screenshot 2023-05-12 at 12 17 36" src="https://github.com/mi6/ic-ui-kit/assets/113986285/50d52613-0ea9-4b86-8c09-b6ca20f165e5"> 

**Screenshot 2:**
<img width="496" alt="Screenshot 2023-05-12 at 09 51 56" src="https://github.com/mi6/ic-ui-kit/assets/113986285/d3c89141-9631-47f3-9554-eaa0c2cafbc0">